platform/orion/core/generic/vault/vault.go

```diff
@@ -146,8 +145,7 @@ func (db *Vault) CommitTX(txid string, block uint64, indexInBloc int) error {
 	db.storeLock.Lock()
 	defer db.storeLock.Unlock()
 
-	m, _ := json.Marshal(i.rws)
-	logger.Debugf("committing \n[%s]\n", string(m))
+	logger.Debugf("committing \n[%s]\n", i.rws.String())
```

Comment:
since the implementation of the db driver will also use json Marshal on the rws later, this here was just for debugging purpose. 
If logging this rws as json is desired this should happen only in a `if logger.Level() == DEBUG` case